proposals/nnnn-template-additional-submenus.md

```diff
@@ -0,0 +1,86 @@
+# Template Improvements: Additional Sub-Menus
```

Comment:
Aside from the Pull Request name and Proposal title, "submenu" is one word (without the hyphen) throughout the document.  I think the PR name and Proposal title should be updated to match.